go-sdk/pkg/sdk/keptn.go

```diff
@@ -314,3 +317,19 @@ func (k *Keptn) createErrorFinishedEventForTriggeredEvent(event cloudevents.Even
 	c.SetData(cloudevents.ApplicationJSON, commonEventData)
 	return c
 }
+
+// storing wait group into context to sync before shutdown
```

Comment:
pls provide a proper formatted comment starting with the function name like "// getGracefulContext is creating a context containing....."